### DISCUSSION TITLE
Questions regarding to the association.

### DISCUSSION THREAD

**Discussion by R0B1NNN1** [stefanbschneider/mobile-env#45] [Q&A]
@stefanbschneider :

Hi, I have a question about the `mobile-env`, is there any association between BS and UE? Like any BS can connect to the UE or they can only connect to the UE in its own range. After the UE are out of range, the BS will not serve it anymore?

**Comment by stefanbschneider** [comment]
Hi @R0B1NNN1 , BS and UEs can only connect if they are in range and the connection is dropped when a UE moves out of range.

Specifically, the SNR needs to be above a predefined SNR threshold to allow a connection: https://github.com/stefanbschneider/mobile-env/blob/main/mobile_env/core/base.py#L263

The SNR threshold can be defined per UE and is given as `snr_tr` argument in the UE constructor: https://github.com/stefanbschneider/mobile-env/blob/main/mobile_env/core/entities.py#L38C9-L38C15

Does this answer your question?
